packages/IMAPClient-UI.package/ICFolderDialog.class/instance/emailListKeyPressMapping.st

```diff
@@ -0,0 +1,6 @@
+events
+emailListKeyPressMapping
+
+	^ {
+		self class defaultOpenEmailChar -> [self selectedEmail ifNotNil: [self openEmail: self selectedEmail]]
```

Comment:
maybe have the `$o` here directly so the functionality "Press o to open email" is visible in this line directly?!

Reply:
Done